### DISCUSSION TITLE
Logo design

### DISCUSSION THREAD

**Discussion by NNBnh** [helix-editor/helix#946] [Ideas] [answered]
Hello,
I'm a Kakoune user and just discover Helix editor yesterday but I already fall in love with it features. I want to contribute a bit for this project and realize that this organization/editor dose not have a logo yet, so i attempt to make one:

**Design decision:**
- As a non native English speaker, at first i didn't know what a Helix is... But after looking up the [Helix's Wikipedia page](https://wikiless.org/wiki/Helix?lang=en), seem like it's a spiral (hope I'm correct) and so is the shape of the logo.
- The reason i chose purple is because it's the default theme color as well as the [Helix website](https://helix-editor.com)'s background.

**Result:**
This logo was heavily inspired by [VScode's logo](https://code.visualstudio.com/brand) with [Material color](https://material.io/design/color/the-color-system.html):

|Defaults|Rotated|Polish|
|-|-|-|
|![1](https://user-images.githubusercontent.com/43980777/139640199-1cb4d677-a9ec-4d4f-a317-ff604c4c1376.png)|![2](https://user-images.githubusercontent.com/43980777/139640229-81e2bc2f-a7d3-4621-bc5d-ec4a9c4b696f.png)|![3](https://user-images.githubusercontent.com/43980777/139695640-1cfa3aaa-2ef4-40c9-af37-3b177338af5b.png)|

> *[Figma link.](https://www.figma.com/file/QNyHTXJhq4gdhfnzUCePlq/Helix?node-id=0%3A1)*

Probably need some polish, not sure how thought...

- **Update 1:**
  - I realize that we could make the logo look like the letter H ("H" stand for Helix of course).
- **Update 2:**
  - Add polish ver.
  - Move focus to #283.

**Comment by archseer** [comment, marked as answer]
Wow, I quite like this! There was a [previous thread regarding logos](https://github.com/helix-editor/helix/issues/283), take a look and maybe you can get some inspiration from there? I like the design but it would be cool if we got it to look more like a [double helix](https://wikiless.org/wiki/Double_helix?lang=en) (DNA). ([Some similar icons, but these are 2D](https://thenounproject.com/search/?q=helix))

also \cc ing @pickfire

**Reply by pickfire** [reply]
Nice. I like the color and shape. But compared to the [previous nice logo contributed by @DrZingo](https://github.com/helix-editor/helix/issues/283#issuecomment-885779814) which gives the context of "hx" this does not, other than that the design looks nice. By the way, we are probably in a similar timezone.

**Reply by kirawi** [reply]
I actually quite prefer this. Even though it doesn't spell out Helix, it's unique and aesthetically pleasing.

**Reply by Omnikar** [reply]
This does look cool, but I myself prefer DrZingo's designs.

**Reply by AnzenKodo** [reply]
I also like it & I agree with @kirawi this does look unique than others.

**Reply by kljensen** [reply]
I agree that the @DrZingo logo provides a little more "helix" and "hx" than the design from @NNBnh. The latter has more polish and better color. (Though, I think wiser to avoid drop shadows in the logo.)

**Comment by hovsater** [comment]
This is beautiful. I really like this one. ❤️
